<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoreSettingsForCopy</class>
 <widget class="QDialog" name="MoreSettingsForCopy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MoreSettingsForCopy{

	background-image: url(:/Images/moreSettings_ Copy.png);
	background-repeat: repeat;
}

#MoreSettingsForCopy *{
	border-image: url();
}</string>
  </property>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>325</y>
     <width>31</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ResStr_On</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="docSizeList">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>160</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">
#docSizeList {
background: transparent;  
}
</string>
   </property>
   <item>
    <property name="text">
     <string>ResStr_A4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ResStr_A5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ResStr_B5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ResStr_Letter</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ResStr_Executive</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>203</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ResStr_Output_Size</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>356</x>
     <y>274</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ResStr_Tips_for_Copying</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="bt4in1">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>323</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Images/4in1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bt2in1">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>323</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Images/2in1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>313</y>
     <width>80</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
#label_21 {
	background-color: rgb(198, 198, 198);
	border-radius:8px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btAdd">
   <property name="geometry">
    <rect>
     <x>204</x>
     <y>76</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
#btAdd {
	border-image: url(:/Images/Btn_BackGround.png);
}
#btAdd:pressed {
	border-image: url(:/Images/Btn_ClickBackGround.png);
}
#btAdd:hover {
	color: rgb(0, 170, 0);
}</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>202</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ResStr_Media_Type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="bt9in1">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>323</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Images/9in1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btClose">
   <property name="geometry">
    <rect>
     <x>506</x>
     <y>4</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">
#btClose {
	border-image: url(:/Images/MainWindow_Close.PNG);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btID">
   <property name="geometry">
    <rect>
     <x>416</x>
     <y>322</y>
     <width>40</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
#btID {
	border-image: url(:/Images/CheckBox_Open.png);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="paperTypeList">
   <property name="geometry">
    <rect>
     <x>332</x>
     <y>232</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#paperTypeList {
background: transparent;  
}</string>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>ResStr_Plain_Paper</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ResStr_Recycled_Paper</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ResStr_Thick_Paper</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ResStr_Thin_Paper</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ResStr_Label</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>360</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ResStr_On</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>131</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ResStr_Document_Size</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>380</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>ResStr_2_in_1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>313</y>
     <width>80</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
#label_20 {
	background-color: rgb(198, 198, 198);
	border-radius:8px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_32">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>227</x>
     <y>380</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>ResStr_9_in_1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>39</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ResStr_Scaling</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_tip">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>105</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(176, 255, 176);</string>
   </property>
   <property name="text">
    <string>ResStr_The_valid_range_is_25_400__please_confirm_and_enter_again_</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btDefault">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>429</y>
     <width>121</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
#btDefault {
	border-image: url(:/Images/Btn_Gray.png);
	font: 14pt &quot;微软雅黑&quot;;
}
#btDefault:pressed {
	border-image: url(:/Images/Btn_Gray_Pressed.png);
	
	color: rgb(255, 255, 255);
}
#btDefault:hover {
	color: rgb(0, 170, 0);
}</string>
   </property>
   <property name="text">
    <string>ResStr_Default</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="scaling">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>73</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
#scaling {
    border: 2px transparent;  
}</string>
   </property>
   <property name="text">
    <string>132</string>
   </property>
   <property name="maxLength">
    <number>4</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btNInOne">
   <property name="geometry">
    <rect>
     <x>416</x>
     <y>359</y>
     <width>40</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
#btNInOne{
	border-image: url(:/Images/CheckBox_Open.png);
} </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>223</x>
     <y>313</y>
     <width>80</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
#label_22 {
	background-color: rgb(198, 198, 198);
	border-radius:8px;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>132</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ResStr_DPI</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btReduce">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>76</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>7</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
#btReduce {
	border-image: url(:/Images/Btn_BackGround.png);
}
#btReduce:pressed {
	border-image: url(:/Images/Btn_ClickBackGround.png);
}
#btReduce:hover {
	color: rgb(0, 170, 0);
}</string>
   </property>
   <property name="text">
    <string>—</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ResStr_Document_Type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="isNinOne">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>274</y>
     <width>161</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QRadioButton::indicator::unchecked {
	image: url(:/Images/Uncheck.png);
}

QRadioButton::indicator::checked {
	image: url(:/Images/Checked.png);
}</string>
   </property>
   <property name="text">
    <string>ResStr_N_in_1_Copy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btOK">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>429</y>
     <width>121</width>
     <height>35</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">
#btOK {
	border-image: url(:/Images/Btn_Green.png);
	font: 14pt &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
}

#btOK:pressed {
	color: rgb(255, 255, 255);
	border-image: url(:/Images/Btn_Green_Pressed.png);
	font: 14pt &quot;微软雅黑&quot;;	

}

#btOK:hover {
	font: 14pt &quot;微软雅黑&quot;;	
	color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>ResStr_OK</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>3</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>ResStr_More_Settings</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_31">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>380</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>ResStr_4_in_1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>386</x>
     <y>323</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ResStr_Off</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="dpiList">
   <property name="geometry">
    <rect>
     <x>332</x>
     <y>159</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#dpiList {
background: transparent;  
}
</string>
   </property>
   <item>
    <property name="text">
     <string>300 × 300 dpi</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>600 × 600 dpi</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>323</y>
     <width>51</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ResStr_ID_Card</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>73</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QComboBox" name="outPutSizeList">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>232</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#outPutSizeList {
	background: transparent;  
}</string>
   </property>
   <item>
    <property name="text">
     <string>ResStr_Letter</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ResStr_A4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ResStr_A5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ResStr_A6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ResStr_B5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ResStr_B6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ResStr_Executive</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ResStr_16K</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>361</y>
     <width>51</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ResStr_N_in_1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>286</x>
     <y>69</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
border: transparent;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="btText">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>9</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QRadioButton::indicator::unchecked {
	image: url(:/Images/Uncheck.png);
}

QRadioButton::indicator::checked {
	image: url(:/Images/Checked.png);
}</string>
    </property>
    <property name="text">
     <string>ResStr_Text</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btPicture">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">
QRadioButton::indicator::unchecked {
	image: url(:/Images/Uncheck.png);
}

QRadioButton::indicator::checked {
	image: url(:/Images/Checked.png);
}</string>
    </property>
    <property name="text">
     <string>ResStr_Photo</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>386</x>
     <y>360</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ResStr_Off</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
